lib/irb.rb

```diff
@@ -993,12 +988,8 @@ def run(conf = IRB.conf)
         trap("SIGINT", prev_trap)
         conf[:AT_EXIT].each{|hook| hook.call}
 
-        if forced_exit
-          context.io.save_history if supports_history_saving
-          Kernel.exit(0)
-        else
-          context.io.save_history if save_history
-        end
+        context.io.save_history if save_history
+        Kernel.exit(0) if forced_exit
```

Comment:
Recommend `exit` or `exit(true)` instead of `exit(0)`.

Reply:
Thank you. Address it in https://github.com/ruby/irb/pull/868